docs/usage.rst

```diff
@@ -0,0 +1,188 @@
+   This client has the following magic-folders:
+     default: /home/alice/Work/Magic
+       magic: /home/alice/Documents/Shared
+
+
+Inviting Participant Devices
+----------------------------
+
+A new participant device is invited to collaborate on a magic folder
+using the ``magic-folder invite`` command:
+
+
+.. code-block:: console
+
+   $ magic-folder --node-device=alice invite magic: bob
+
+An invitation code is created using an alias for an existing magic
+folder (``magic:`` above) and a nickname for the new participant
+device (``bob`` above).  The magic folder alias identifies a
+previously created magic folder.  The nickname is assigned to the
+participant device in the magic folder configuration and grid state.
+Note that only the creator of a magic folder can invite new
+participant devices.
+
+Joining a Magic Folder
+----------------------
+
+A participant device accepts an invitation using the ``magic-folder
+join`` command:
+
+.. code-block:: console
+
+   $ magic-folder -d bob join $INVITECODE /home/bob/Documents/Shared
+
+The first argument required is an invitation code, as described in
+`Inviting Participant Devices`_ is required.  The second argument
+required is the path to a local directory.  This is the directory to
+which content will be downloaded and from which it will be uploaded.
+
+Further options are documented in ``magic-folder join --help``.
+
+Leaving a Magic Folder
+----------------------
+
+A participant device can reverse the action of joining a magic folder
+using the ``magic-folder leave`` command.
+
+The only option which can be supplied (but which has a default) is the
+nickname of the magic folder to leave.  Once a device has left a magic
+folder, further changes to files in the folder will not be
+synchronized.  The local directory is not removed.
+
+See ``magic-folder leave --help`` for details.
+
+.. _config-file:
+
+Magic Folder configuration file
+-------------------------------
+
+The commands documented above manipulate ``magic_folders.yaml`` in the
+Tahoe-LAFS node's private area.  This is a historical artifact
+resulting from the origin of Magic Folder as a part of Tahoe-LAFS
+itself. Configuration can be changed by modifying this file directly.
+
+Tahoe-LAFS also has historical configuration for Magic-Folder in the
+``tahoe.cfg`` configuration file.  This configuration is deprecated.
+In particular, the ``enabled`` boolean in the ``magic_folder`` section
+is ignored by Magic-Folder.  It should be set to false to prevent any
+Magic-Folder functionality included in Tahoe-LAFS from activating.  To
+activate the Magic-Folder configuration for a Tahoe-LAFS node, use
+``magic-folder run``.
+
+
+A quick test
+------------
+
+If you want to test that things work as expected using a single
+machine, you can create two separate Tahoe-LAFS nodes, and assign
+corresponding magic folders with them, like so:
+
+.. code-block:: console
+
+   $ ALICE_NODE=../grid/alice
+   $ ALICE_FOLDER=../local/alice
+
```

Comment:
I think these will need `export ALICE_NODE=...` if people are copy-pasting into a shell (instead of putting this all in one shell-script) right? At least, for bash.

Reply:
Not sure `export` would be necessary, since no sub-shells would be involved here?  `$ALICE_NODE` should expand in the same bash process.